src/runtime-rs/crates/runtimes/common/src/types/mod.rs

```diff
@@ -0,0 +1,209 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+mod trans_from_shim;
+mod trans_into_shim;
+
+use std::fmt;
+
+use kata_types::mount::Mount;
+
+#[derive(Debug, Clone)]
+pub enum Request {
```

Comment:
I wonder if it 's worth adding a comment on this to say that any change to `Request` should also update `Response`. It should be obvious, but...

Reply:
Thanks for your review, already add a comment.